SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -187,6 +187,8 @@ void ComponentReader::ApplySchemaObject(Schema_Object* ComponentObject, UObject*
 		}
 	}
 
+	Channel->RemoveRepNotifiesWithUnresolvedObjs(RepNotifies, *Replicator.RepLayout, RootObjectReferencesMap, Object);
```

Comment:
Since `RemoveRepNotifiesWithUnresolvedObjs` is touching every rep notify, is it possible to check for unresolved objects in place? Or do we only know what is unresolved after the entire update has been processed?

Reply:
Yeah, as fields are resolved separately, and multiple fields could correspond to a single repnotify, we need to wait until they're all fired before cancelling the callback.